RecoLocalCalo/HGCalRecProducers/interface/HGCalImagingAlgo.h

```diff
@@ -48,7 +48,7 @@ class HGCalImagingAlgo : public HGCalClusteringAlgoBase
      fcPerMip_(ps.getParameter<std::vector<double> >("fcPerMip")),
      fcPerEle_(ps.getParameter<double>("fcPerEle")),
      nonAgedNoises_(ps.getParameter<edm::ParameterSet>("noises").getParameter<std::vector<double> >("values")),
-     noiseMip_(ps.getParameter<edm::ParameterSet>("noiseMip").getParameter<double>("value")),
+     noiseMip_(ps.getParameter<edm::ParameterSet>("noiseMip").getParameter<double>("noise_MIP")),
```

Comment:
there are no matching changes in ```fillPSetDescription```. I guess this algo is broken then by this change (at least in a limited scope of using fillPSetDescription).